orc8r/cloud/deploy/orc8r_deployer/docker/run_deployer

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
```

Comment:
Name this file `run_deployer.bash`

Reply:
I'm not in favor of really adding a bash extension. I feel it just makes the name of the script ugly and doesn't serve any purpose. Just a personal choice. It seems more succinct to keep this as run_deployer.